stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,88 @@
+[id="proc-aap-gcp-application-upgrade"]
+
+Technical Preview
+~~~~~~~~~~~~~~~~~
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. The upgrade process covers upgrading of automation hub and automation controller. The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. All upgrades should be no more than one major version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonGCP} to 2.2.20230215, you must be on the first version of {AAPonGCP}.
+=====
+
+.Prerequisites
+* You require a docker installed in order to run the upgrade playbook.
+
+== Upgrade Process
+
+The following instructions describe how to run the upgrade playbook.
```

Comment:
<try to suggest a file structure to the customer. This may help to prevent confusion.>